pkg/bindings/containers/archive.go

```diff
@@ -49,13 +50,16 @@ func Stat(ctx context.Context, nameOrID string, path string) (*entities.Containe
 	return statReport, finalErr
 }
 
-func CopyFromArchive(ctx context.Context, nameOrID string, path string, reader io.Reader) (entities.ContainerCopyFunc, error) {
+func CopyFromArchive(ctx context.Context, nameOrID string, path string, reader io.Reader, options *CopyOptions) (entities.ContainerCopyFunc, error) {
 	conn, err := bindings.GetClient(ctx)
 	if err != nil {
 		return nil, err
 	}
 	params := url.Values{}
 	params.Set("path", path)
+	if options != nil && options.Chown != nil {
+		params.Set("copyUIDGID", strconv.FormatBool(*options.Chown))
+	}
```

Comment:
Can you use `params.WithChown(options.Chown)` (see other bindings)?

Reply:
@vrothberg don't you mean to use `ToParams()` ?